lib/layout/dummyLayoutV2.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * dummyLayoutV2.js
+ *
+ * this creates a basic poorly laid out graph
+ *
+ * @author John O'Brien.
+ */
+
+// Create a graph instance.
+
+function buildGraph(graph) {
+  var lib = './../';
```

Comment:
Noted - I've made similar fixes to the branch I PR'd early, but I have yet to push out. My mistake.